streaming/src/test/run_streaming_queue_test.sh

```diff
@@ -44,15 +44,22 @@ if [ ! -d "$RAY_ROOT/python" ]; then
   exit 1
 fi
 
-REDIS_MODULE="./bazel-bin/libray_redis_module.so"
-REDIS_SERVER_EXEC="./bazel-bin/external/com_github_antirez_redis/redis-server"
-STORE_EXEC="./bazel-bin/plasma_store_server"
-REDIS_CLIENT_EXEC="./bazel-bin/redis-cli"
-RAYLET_EXEC="./bazel-bin/raylet"
-STREAMING_TEST_WORKER_EXEC="./bazel-bin/streaming/streaming_test_worker"
-GCS_SERVER_EXEC="./bazel-bin/gcs_server"
-
-# Allow cleanup commands to fail.
+REDIS_MODULE="$RAY_ROOT/bazel-bin/libray_redis_module.so"
+REDIS_SERVER_EXEC="$RAY_ROOT/bazel-bin/external/com_github_antirez_redis/redis-server"
+STORE_EXEC="$RAY_ROOT/bazel-bin/plasma_store_server"
+REDIS_CLIENT_EXEC="$RAY_ROOT/bazel-bin/redis-cli"
+RAYLET_EXEC="$RAY_ROOT/bazel-bin/raylet"
+STREAMING_TEST_WORKER_EXEC="$RAY_ROOT/bazel-bin/streaming/streaming_test_worker"
+GCS_SERVER_EXEC="$RAY_ROOT/bazel-bin/gcs_server"
+
+# clear env
+pgrep "plasma|DefaultDriver|DefaultWorker|AppStarter|redis|http_server|job_agent" | xargs kill -9 &> /dev/null
+
 # Run tests.
-./bazel-bin/streaming/streaming_queue_tests $STORE_EXEC $RAYLET_EXEC "$RAYLET_PORT" $STREAMING_TEST_WORKER_EXEC $GCS_SERVER_EXEC $REDIS_SERVER_EXEC $REDIS_MODULE $REDIS_CLIENT_EXEC
+
+# to run specific test, add --gtest_filter, below is an example
```

Comment:
Remove if unused.

Reply:
I think it's necessary to add an example here.